_includes/search.html

```diff
@@ -0,0 +1,25 @@
+<script language="Javascript" type="text/javascript">
+  function my_search_google()
+  {
+    var query = document.getElementById("my-google-search").value;
+    window.open("http://google.com/search?q=" + query
+	+ "%20site:" + "http://theforeman.org");
+  }
+</script>
+
+<div class="container container-search">
+  <a class="pull-left" href="/feed.xml">
+    <img src="/static/images/rss.png" alt="RSS Feed" />
+    Get the RSS News Feed
```

Comment:
I'm happy to change the wording, but the icon alone is easily missed (especially as it's not visible anywhere else), and with the search bar to the right we have the space for *something*. Suggestions welcome for improvements welcome - I wasn't 100% happy with it either.